frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/ReSchedulHearing.js

```diff
@@ -139,7 +126,27 @@ const RescheduleHearing = ({ onSelect, onCancel, onDismiss }) => {
           <strong>{t("RESCHEDULE_HEARING_MESSAGE")}</strong>
         </CardText>
       </div>
-      {children}
+      {rescheduleAll?(
+          <div>
+          <div key="5">Hearing Type</div>
+          <div key="6">No. of Hearings</div>
+          <div key="7">Initial Hearing Date</div>
+          <div key="8">Initial Time Slot</div>
+          </div>
+      ):(
+        <div>
+          <div key="1">Case Number: {t(`${a}`)}</div>
```

Comment:
a number doesn't have to be internationalized
the text to the left of the number can be however